DOCAPI-7437: os_thread_priority docs (#6883)
ClickHouse/ClickHouse: docs/en/operations/settings/settings.md

- [Table engine Distributed](../../operations/table_engines/distributed.md)
- [`distributed_replica_error_half_life`](#settings-distributed_replica_error_half_life)

## os_thread_priority {#setting-os_thread_priority}

> Sets the priority ([nice](https://en.wikipedia.org/wiki/Nice_(Unix))) for threads that process queries. OS planner uses this priority when choosing the thread for execution.

Reviewer: ```suggestion
Sets the priority ([nice](https://en.wikipedia.org/wiki/Nice_(Unix))) for threads that execute queries. OS scheduler considers this priority when choosing the next thread to run on each available CPU core.
```

!!! warning "Warning"

>     To use this setting, you need to set the `CAP_SYS_NICE` capability. The `clickHouse-server` package set it up when installing. Some virtual environments don't allow to set the `CAP_SYS_NICE` capability. In this case `clickhouse-server` shows an informational message at the start.

Reviewer: ```suggestion
    To use this setting, you need to set the `CAP_SYS_NICE` capability. The `clickhouse-server` package sets it up during installation. Some virtual environments don't allow to set the `CAP_SYS_NICE` capability. In this case `clickhouse-server` shows a message about it at the start.
```

Possible values:

You can set values in the `[-20, 19]` range.

> The less value has a higher priority, so, threads with low values of `nice` priority are executed more frequently then threads with high values. Higher values are preferable for non-interactive and for long executing queries because it allows the server to stay available for short queries.

Reviewer: ```suggestion
The lower value means a higher priority. Threads with low values of `nice` priority are executed more frequently than threads with high values. High values are preferable for long running non-interactive queries because it allows them to quickly give up resources in favour of short interactive queries when they arrive.
```
